webui/react/src/pages/ExperimentList.table.tsx

```diff
@@ -1,26 +1,20 @@
-import { ColumnsType } from 'antd/lib/table';
+import { ColumnType } from 'antd/lib/table';
 import React from 'react';
 
 import {
-  actionsRenderer, experimentDescriptionRenderer, experimentProgressRenderer,
+  actionsRenderer, experimentProgressRenderer,
   expermentDurationRenderer, relativeTimeRenderer, stateRenderer, userRenderer,
 } from 'components/Table';
 import { ExperimentItem } from 'types';
 import { alphanumericSorter, runStateSorter, stringTimeSorter } from 'utils/data';
 import { experimentDuration } from 'utils/time';
 
-export const columns: ColumnsType<ExperimentItem> = [
+export const columns: ColumnType<ExperimentItem>[] = [
```

Comment:
**question:** is one better than the other (`ColumnsType` vs `ColumnType`)? Let's pick one and use that consistently. I think I like the change you have here because it's more obvious that it's a list of columns because of the `[]` suffix. Plus it's easy to miss the `s` in `ColumnsType`.

Reply:
here I needed it to be known as a list to typescript explicitly so I could do list operations on it. sounds good.

Reply:
i updated the other columnstype